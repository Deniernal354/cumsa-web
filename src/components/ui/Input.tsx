import type { InputHTMLAttributes } from "react";

type Props = InputHTMLAttributes<HTMLInputElement> & { className?: string };

export default function Input({ className = "", ...props }: Props) {
  return (
    <input
      className={`w-full rounded-md border border-black/10 bg-[var(--surface)] px-3 py-2 text-[var(--foreground)] placeholder-[var(--muted)] focus:outline-none focus:ring-2 focus:ring-[var(--blue)] ${className}`}
      {...props}
    />
  );
}
