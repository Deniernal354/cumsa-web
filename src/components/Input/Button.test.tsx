import { render } from "@testing-library/react";
import Input from "./Input";

test("Input snapshot", () => {
  const { asFragment } = render(<Input />);
  expect(asFragment()).toMatchSnapshot();
});
