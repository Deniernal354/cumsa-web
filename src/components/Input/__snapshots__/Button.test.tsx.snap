// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input snapshot 1`] = `
<DocumentFragment>
  <input
    class="w-full rounded-md border border-black/10 bg-[var(--surface)] px-3 py-2 text-[var(--foreground)] placeholder-[var(--muted)] focus:outline-none focus:ring-2 focus:ring-[var(--blue)] "
  />
</DocumentFragment>
`;
